dom0-updates/qubes-dom0-update

```diff
@@ -65,15 +65,15 @@ done
 # but do allow explicit template reinstalls
 if [ "$YUM_ACTION" == "reinstall" ] && [[ "$PKGS" == *"qubes-template-"* ]]; then
     TEMPLATE_EXCLUDE_OPTS=""
-    echo "WARNING: Reinstalling a template will erase files in /home and /rw !"
+    echo "WARNING: Reinstalling a template will erase all files in template's /home and /rw !"
     
-    $ONEPKG=`cut -f 1 -d ' ' <<<$PKGS`
+    ONEPKG=`cut -f 1 -d ' ' <<<$PKGS`
     if [[ "$ONEPKG" == "qubes-template-"* ]] && [[ "$ONEPKG" == "${PKGS#\ }" ]]; then # test "$PKGS" minus space
     # Prepare to backup template root.img in case reinstall doesn't complete.
         TEMPLATE=${ONEPKG#qubes-template-}
-        TEMPLATE_NETVM=`qvm-prefs $TEMPLATE netvm` || exit 1
-        if [[ "$TEMPLATE_NETVM" == *"(default)" ]] ; then TEMPLATE_NETVM="default"
-        BAK_TEMPLATE_ROOT=`qvm-prefs $TEMPLATE root_img` || exit 1
+        TEMPLATE_NETVM=`qvm-prefs --force-root $TEMPLATE netvm` || exit 1
+        [[ "$TEMPLATE_NETVM" == *"(default)" ]] && TEMPLATE_NETVM="default"
```

Comment:
Better restore `if` (and fix syntax). Otherwise if someone would add `set -e` in the future, this would fail.